<launch>
   <node type="raspicam_node" pkg="raspicam_node" name="raspicam_node" output="screen">
 
    <!-- <param name="camera_info_url" value="package://raspicam_node/camera_info/camerav2_1280x720.yaml"/> -->
    <param name="camera_name" value="front_cam"/>
    <param name="width" value="320"/>
    <param name="height" value="240"/>
    <!-- We are running at 90fps to reduce motion blur -->
    <param name="framerate" value="90"/>

    <param name="camera_frame_id" value="front_cam"/>

  </node>

  <node type="cubos_node.py" pkg="robocar" name="cubos" output="screen">
    
    <!-- <param name="camera_info_url" value="package://raspicam_node/camera_info/camerav2_1280x720.yaml"/> -->
    <param name="img_topic" value="/raspicam_node/image/compressed"/>

  </node>
</launch>